plugins/org.eclipse.n4js.cli/src/org/eclipse/n4js/cli/N4jscOptionsValidater.java

```diff
@@ -174,10 +174,18 @@ private static void validateTestCatalogFile(N4jscOptions options) throws N4jscEx
 		}
 	}
 
-	private static void validatePerformanceReport(N4jscOptions options) throws N4jscException {
-		if (options.getPerformanceReport() != null && Strings.isNullOrEmpty(options.getPerformanceKey())) {
+	private static void validatePerformanceOptions(N4jscOptions options) throws N4jscException {
+		if (options.getPerformanceReport() == null) {
+			String msg = "Missing performance report.";
+			throw new N4jscException(N4jscExitCode.OPTION_INVALID, msg);
```

Comment:
Do we have a test for this (if possible)?

Reply:
A value of `null` cannot be produced at the moment. I only added this (1) for consistency with the case "Missing performance key" right below this if-block and (2) in case we change the default in the future. So we cannot create a test for this, for now.